### PR TITLE
Flight recorder improvements

### PR DESCRIPTION
**Pull request by johanandren**
A small app to dump AFR-files from somewhere else.

Merged lists of alerts, hiFreq and loFreq events into one.

### CODE REVIEW

**Comment by patriknw** [comment]
LGTM

File: akka-remote/src/main/scala/akka/remote/artery/FlightRecorderReader.scala [hunk 1]
```diff
@@ -72,16 +83,15 @@ private[akka] object FlightRecorderReader {
       raFile = new RandomAccessFile(flightRecorderFile.toFile, "rw")
       channel = raFile.getChannel
       reader = new FlightRecorderReader(channel)
-      println(reader.structure)
-
-      println("--- ALERT ENTRIES")
-      reader.structure.alertLog.logs.foreach(log ⇒ println(log.richEntries.mkString("\n")))
+      val alerts: Seq[FlightRecorderReader#Entry] = reader.structure.alertLog.logs.flatMap(_.richEntries)
```
Comment (drewhk) [inline akka-remote/src/main/scala/akka/remote/artery/FlightRecorderReader.scala]: shouldn't filter for active logs? Or logs does that already?

File: akka-remote/src/main/scala/akka/remote/artery/FlightRecorderReader.scala [hunk 2]
```diff
@@ -72,16 +83,15 @@ private[akka] object FlightRecorderReader {
...
+      val hiFreq: Seq[FlightRecorderReader#Entry] = reader.structure.hiFreqLog.logs.flatMap(_.compactEntries)
+      val loFreq: Seq[FlightRecorderReader#Entry] = reader.structure.loFreqLog.logs.flatMap(_.richEntries)
 
-      println("--- HI FREQUENCY ENTRIES")
-      reader.structure.hiFreqLog.logs.foreach(log ⇒ println(log.compactEntries.mkString("\n")))
+      implicit val ordering = Ordering.fromLessThan[FlightRecorderReader#Entry]((a, b) ⇒ a.timeStamp.isBefore(b.timeStamp))
+      val sorted = SortedSet[FlightRecorderReader#Entry](alerts: _*) ++ hiFreq ++ loFreq
```
Comment (drewhk) [inline akka-remote/src/main/scala/akka/remote/artery/FlightRecorderReader.scala]: hm, does it need to be a Set?

**Review by drewhk — APPROVED** [review]

**Comment by drewhk** [comment]
LGTM

File: akka-remote/src/main/scala/akka/remote/artery/FlightRecorderReader.scala (hunk 1 above)
Comment (johanandren) [inline akka-remote/src/main/scala/akka/remote/artery/FlightRecorderReader.scala]: I haven't actually seen a case where there is data in more than one log, so I'm not sure how it works.

File: akka-remote/src/main/scala/akka/remote/artery/FlightRecorderReader.scala (hunk 2 above)
Comment (johanandren) [inline akka-remote/src/main/scala/akka/remote/artery/FlightRecorderReader.scala]: Can probably be replaced with something more performant, didn't want to spend too much time on it so I just used the first thing coming to my mind.

**Pull request merged**